config/redirects

```diff
@@ -17,4 +17,5 @@ raw: ${prefix}/stable -> ${base}/current/
 [*-v5.0]: ${prefix}/${version}/quick-start/download-and-install/ -> ${base}/${version}/quick-start/
 [*-v5.0]: ${prefix}/${version}/quick-start/next-steps/ -> ${base}/${version}/quick-start/
 
-[v4.3-master]: ${prefix}/${version}/fundamentals/utf8-validation -> ${base}/${version}/fundamentals/bson/utf8-validation/
\ No newline at end of file
+[v4.3-master]: ${prefix}/${version}/fundamentals/utf8-validation -> ${base}/${version}/fundamentals/bson/utf8-validation/
+[*-v4.2]: ${prefix}/${version}/fundamentals/utf8-validation -> ${base}/${version}/
```

Comment:
Suggestion:
For uniformity (would need to investigate whether there is a functional reason), the urls should have a trailing slash. Mut might do this automatically, but just thought uniformity might raise fewer questions.